src/gettingstarted/local/docksal.md

```diff
@@ -0,0 +1,72 @@
+# Using Docksal for local development
+
+[Docksal](https://docksal.io) is a docker based local development tool that plays nicely with Platform.sh.  It is maintained by a community of developers but is a viable option for most Platform.sh projects.
+
+See the [Docksal documentation](https://docs.docksal.io/) for installing and setting up Docksal on your system.
+
+Docksal will ask you to create a `.docksal` directory in your application root, which functions similarly to the `.platform.app.yaml` file. It is safe to check this directory into your Git repository as Platform.sh will simply ignore it.
+
+All interaction with a Docksal project will happen through the `fin` command. For more info you can type the `help` subcommand to get all available commands and options.
+
+```bash
+fin help
+```
+
+## Using Platform.sh CLI within Docksal
+
+To start using the Platform.sh CLI within your docksal project. The `SECRET_PLATFORMSH_CLI_TOKEN` will need to be set. This will allow for you to interact with your Platform.sh account without the need to download anything locally.
+
+```bash
+fin config set --global SECRET_PLATFORMSH_CLI_TOKEN=XXX
+```
+
+## Pulling a Platform.sh project
+
+The Docsal CLI images ship with the Platform.sh CLI tool. To use the tool and pull a project locally, make sure you have uploaded your SSH key to your Platform.sh account. Once that has been done and a `SECRET_PLATFORMSH_CLI_TOKEN` has been added using the above step. The following command can be ran. Replace `PROJECT_ID` with your project's ID, which can found within the Platform.sh dashboard. Replace `PROJECT_DIRECTORY` with the name of the local directory you'd like the project cloned into. With your SSH key and the `SECRET_PLATFORMSH_CLI_TOKEN` set, you can execute the following command.
```

Comment:
"Once that has been done and a `SECRET_PLATFORMSH_CLI_TOKEN` has been added using the above step. The following command can be ran."

That's a sentence fragment.  The period and capital before "The following command" can be eliminated entirely.